site/content/en/docs/concepts/workload.md

```diff
@@ -151,6 +151,10 @@ status:
 When a Workload deactivated by Sequential Admission with Ready Pods is re-activated, 
 the requeueState (`.status.requeueState`) will be reset to null.
 
+## Copying labels
+At Workload creation, Kueue has a functionality to copy labels into the Workload from the underlying Job or Pod objects. You can specify which labels should be copied by setting `labelKeysToCopy` field in the configuration API (under `Integrations`).
```

Comment:
Could we mention the default behavior something like this?

```
By default, Kueue copies all Job or Pod labels into the Workload.
```

Reply:
Actually, it copies none :)
But yes, we should add the default behavior.

Reply:
Oops, I am misunderstanding the default behavior. 
Thanks!

Reply:
Added an explanation of the default